spec/requests/discourse_code_review/repos_controller_spec.rb

```diff
@@ -125,9 +126,19 @@ def set_client(client)
         end
 
         it "says yes" do
-          get '/admin/plugins/code-review/organizations/org/repos/repo/has-configured-webhook.json'
+          get "/admin/plugins/code-review/organizations/org/repos/#{repo_name}/has-configured-webhook.json"
           expect(JSON.parse(response.body)).to eq('has_configured_webhook' => true)
         end
+
+        context "when repo name has a . in it" do
+          let(:repo_name) { "Some-coolrepo.org" }
```

Comment:
Minor but can we throw in a `%` too?

Reply:
Hmm I guess that should not be allowed...wonder why we allow it in core? Is it for `%20` I guess?

Reply:
I just got rid of %